### ISSUE TITLE
Code action generates invalid source

### ISSUE THREAD

**Issue by nadeeshaan** [ballerina-platform/ballerina-lang#34410]
**Description:**
Please refer the following screen capture,

https://user-images.githubusercontent.com/1329674/146744954-e022c964-e691-4fce-ace4-86892e83c306.mp4

```
import ballerina/sql;
import ballerinax/mysql;

mysql:Client dbClient = check new("");

function test(string command) {
    sql:ParameterizedQuery q = ``;

    dbClient->execute(q);
}

```

**Comment by nadeeshaan** [comment]
When we look at the diagnostic properties, we get the following type information related to the diagnostic, which leads to the error.
<img width="967" alt="Screenshot 2022-01-03 at 12 27 00" src="https://user-images.githubusercontent.com/1329674/147905808-a3818fe6-5c10-48ec-8be3-99d46ec8efd4.png">

**Comment by dulajdilshan** [comment]
The error couldn't be re-produced. The issue seems to be fixed in `Ballerina 2201.0.0 (Swan Lake) Snapshot build 1
`
